classes/opponentPhaseChargeShot.lua

```diff
@@ -4,15 +4,19 @@ local opponentPhaseChargeShot = {}
 opponentPhaseChargeShot.__index = opponentPhaseChargeShot
 
 
+local AIM_TIME = 2
+local CHARGE_TIME = 0.6
+local TOTAL_TIME = AIM_TIME + CHARGE_TIME
+
+
 --------------------
 -- MAIN CALLBACKS --
 --------------------
 
 function opponentPhaseChargeShot.new()
 	local self = classes.opponentBase.new(opponentPhaseChargeShot)
 
-	self.CHARGE_TIME = 3
-	self.shotCooldown = self.CHARGE_TIME
+	self.timer = AIM_TIME / 2
```

Comment:
Already halfway through aim phase - so the first shot comes out sooner than the rest. Feels pretty nice